src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -39,7 +39,7 @@ object GermlineAssemblyCaller {
     var minAlignmentQuality: Int = 30
 
     @Args4jOption(name = "--reference-fasta", required = true, usage = "Local path to a reference FASTA file")
-    var referenceFastaPath: String = null
+    var referenceFastaPath: String = _
```

Comment:
I'd rather be explicit about what we're setting this value to, I don't remember off the top of my head if strings default to null or ""